functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -25,7 +25,8 @@
 
 public class ProjectBuilder {
 
-    private static final String TEST_REPORT_PREFIX = "TEST-";
+    public static final String TEST_REPORT_PREFIX = "TEST-";
+    public static final String IN_PROJECT_DIR = ".reports";
```

Comment:
Why do we call it `IN_PROJECT_DIR`? Could we have a better name for it across the project (also in `InProjectTestReportLoader`)?